### PR TITLE
Add New Board Definition for CircuitPython

### PR DESCRIPTION
**Pull request by hulkco**
Add supprot for Electronic Cats Boards boards "Meow Meow, CatWan USBStick & Bast Pro Mini M0"

### CODE REVIEW

**Comment by sabas1080** [comment]
Please add VID for any board of the project PIDCodes 

`(0x1209, None),    # Any PIDCodes Boards`

pid.codes is a registry of USB PID codes for open source hardware projects. You can see a list of PID assignments, or learn how to get your own.
https://github.com/pidcodes/pidcodes.github.com

File: mu/modes/adafruit.py [hunk 1]
```diff
@@ -57,6 +57,9 @@ class AdafruitMode(MicroPythonMode):
         (0x2B04, 0xC00C),  # Particle Argon
         (0x2B04, 0xC00D),  # Particle Boron
         (0x2B04, 0xC00E),  # Particle Xenon
+        (0xBAB1, 0x1209),  # Electronic Cats Meow Meow
+        (0xBAB2, 0x1209),  # Electronic Cats CatWAN USBStick
+        (0xBAB3, 0x1209),  # Electronic Cats Bast Pro Mini M0
```
[the remark this answers is not in the document]
Comment (hulkco) [reply mu/modes/adafruit.py]: I do not know, I'll ask the guys from Electronic Cats, I just use the data that is in the mpconfigboard.mk file, for each of the cards, When they give me an answer, I will make the changes if necessary, Greetings!

[the remark this answers is not in the document]
Comment (hulkco) [reply mu/modes/adafruit.py]: They already answered me, you were right, they had invested the values, I already made the changes.

[the remark this answers is not in the document]
Comment (carlosperate) [reply mu/modes/adafruit.py]: Great, thanks @hulkco!

File: mu/modes/adafruit.py [hunk 2]
```diff
@@ -57,6 +57,10 @@ class AdafruitMode(MicroPythonMode):
         (0x2B04, 0xC00C),  # Particle Argon
         (0x2B04, 0xC00D),  # Particle Boron
         (0x2B04, 0xC00E),  # Particle Xenon
+        (0x1209, 0xBAB1),  # Electronic Cats Meow Meow
+        (0x1209, 0xBAB2),  # Electronic Cats CatWAN USBStick
+        (0x1209, 0xBAB3),  # Electronic Cats Bast Pro Mini M0
+        (0x1209, None),       # Electronic Cats future Board
```
Comment (carlosperate) [inline mu/modes/adafruit.py]: I believe any open source project can register a PID with the 0x1209 VID, so we shouldn't reserve the entire range for this.

Comment (hulkco) [reply mu/modes/adafruit.py]: ok, then delete and leave only 3 Boards?

Comment (carlosperate) [reply mu/modes/adafruit.py]: Yep :)

**Comment by carlosperate** [comment]
> Please add VID for any board of the project PIDCodes
> 
> `(0x1209, None), # Any PIDCodes Boards`
> 
> pid.codes is a registry of USB PID codes for open source hardware projects. You can see a list of PID assignments, or learn how to get your own.
> https://github.com/pidcodes/pidcodes.github.com

Since any open source project can register a new PID under the same 0x1209 VID I don't think we should reserve all PIDs, as it's quite likely there are other boards (like generic [MicroPython or ESP based boards](https://github.com/pidcodes/pidcodes.github.com/blob/51c99d98d2a714de8a9edf5efe50e870222fabac/1209/ADDA/index.md)) there where this does not apply.

File: mu/modes/adafruit.py (hunk 2 above)
Comment (hulkco) [reply mu/modes/adafruit.py]: Ready @carlosperate !!!

**Comment by sabas1080** [comment]
Thanks @hulkco and @carlosperate

**Comment by sabas1080** [comment]
something new?

**Comment by ntoll** [comment]
@hulkco thanks for the contribution, which I've merged. It'll be in the upcoming alpha.2 release.

**Pull request merged**